scripts/ruby/lib/to_es/vra_to_es/fields.rb

```diff
@@ -120,8 +120,10 @@ def publisher
     get_list(@xpaths["publisher"])
   end
 
-  def recipients
-    # TODO default behavior?
+  def recipient
+    eles = @xml.xpath(@xpaths["recipient"])
+    people = eles.map { |p| { "role" => p["role"], "name" => p.text, "id" => "" } }
```

Comment:
Another style recommendation is to not use explicit `return` when unnecessary (https://github.com/bbatsov/ruby-style-guide#no-explicit-return)
In other words, don't write it unless, for example, you're looping through something trying to find what you want to return and `return` is necessary to end the looping.

So this could just end with:
```ruby
eles.map { |p| { "role" => p["role"], "name" => p.text, "id" => "" } }
```
as that `.map` call returns the array we want the method to return

Reply:
We're gonna need to change like all of my code if we're taking out explicit returns...

Reply:
(I don't mean future code, I mean like literally all of my past code)

Reply:
Sorry I wasn't explicit. I'm not concerned about changing all the existing code and don't want to stop other work for it. Just a note that going forward I'd like to follow that style. That's why I selected "approved" for the review.

Reply:
Huh, I always thought using return explicitly was good